contrib/babelfishpg_common/Makefile

```diff
@@ -35,6 +35,7 @@ OBJS += src/uniqueidentifier.o
 OBJS += src/datetime.o
```

Comment:
PR title and description are not helpful. Let's not have the jira IDs, instead have a meaningful title and description.

Reply:
Done